src/lcd.c

```diff
@@ -168,10 +164,17 @@ void LCD_SendWord(uint8_t type, uint8_t word){
     HC595_Write(LCD_DISABLE | type | low);
 }
 
-void LCD_SendASCIIChar(char* character){
+void LCD_SendASCIIChar(const char* character){
     LCD_SendWord(LCD_DATA, (uint8_t)(*character));
 }
 
+void LCD_SendText(char* text){
+    const char* ch = text;
+    while(*ch){
```

Comment:
What are you thoughts about safety here? We'll run into a problem if `text` is longer than 8 characters (kind of, the cursor will just increment off into positions not mapped on the LCD).

I could put a counter in place to make sure that only 8 character are printed? Not sure about the best approach to take when I've got an unstated condition on this code that goes off into "undefined behavior" territory.